tmt/cli.py

```diff
@@ -690,61 +690,61 @@ def tests_import(
 @tests.command(name='export')
 @click.pass_context
 @filter_options_long
-@click.option(
+@option(
     '-h', '--how', metavar='METHOD', default=_test_export_default, show_default=True,
     help='Output format.',
     type=click.Choice(choices=_test_export_formats))
-@click.option(
+@option(
     '--format', metavar='FORMAT', default=_test_export_default, show_default=True,
     help='Output format. Deprecated, use --how instead.',
```

Comment:
This is also deprecated.

Reply:
As mentioned above, I'd rather have a separate PR for this transition.